python/pkg/rules.bzl

```diff
@@ -96,3 +94,44 @@ def pkg_python_app(
         remap_paths = remap_paths,
         visibility = tar_visibility,
     )
+
+def _pkg_py_unzip_app(
+        name,
+        tar,
+        bindir,
+        libdir,
+        entrypoint = None,
+        mode = "0755",
+        env = None,
+        visibility = None,
+        **kwargs):
+    if kwargs.pop("tar_visibility", None):
+        fail("use 'visibility' not 'tar_visibility'")
+    if kwargs.pop("zip_safe", None):
+        fail("'zip_safe' must be false when using py_unzip")
+    if kwargs.pop("use_exec_wrapper", None) == False:
+        fail("use py_unzip directly instead of disabling 'use_exec_wrapper'")
+
+    exec_wrapper_name = "%s_exec_wrapper" % name
+    exec_wrapper(
+        name = exec_wrapper_name,
+        env = env or get_python_env(),
```

Comment:
should this be `if env is not None` ? are empty dictionaries falsy in starlark?

Reply:
from the [spec](https://github.com/bazelbuild/starlark/blob/master/spec.md#dictionaries)

> A dictionary used in a Boolean context is considered true if it is non-empty.


In this case if `env` is specified it's likely not a `dict` but the results of `get_python_env` which returns a `select` statement. those are also truthy so I don't see a need for an explicit `is not None` check